config/default/foia_wizard.settings.yml

```diff
@@ -60,16 +60,16 @@ messages:
     value: "<ul>\r\n\t<li>Visit the FOIA Library at IRS.gov to see if records are available online.</li>\r\n\t<li>Visit the Routine Access to IRS Records designed to make access quick and easy for many types of IRS records frequently requested.</li>\r\n\t<li>If the records you are seeking are not found in the FOIA Library, submit a FOIA request to the IRS.</li>\r\n</ul>\r\n"
     format: rich_text
   m19:
-    value: "<ul>\r\n\t<li>Contact your local field office. Please use the field office locator to find the office that services your zip code.</li>\r\n\t<li>If you are seeking your earnings records, please follow the instructions to complete Form SSA-7050-F4 available at: <a href=\"https://www.ssa.gov/forms/ssa-7050.pdf\">https://www.ssa.gov/forms/ssa-7050.pdf</a> and mail it to the address on the form along with the applicable fee and other evidentiary documents, as required.</li>\r\n</ul>\r\n\r\n<p>The following agency may also have the records you seek:<br />\r\nSSA</p>\r\n"
+    value: "<p>To find your own social security records, contact your local field office. Please use the <a href=\"https://secure.ssa.gov/ICON/main.jsp\">field office locator</a> that services your zip code.</p>\r\n\r\n<p>You can also submit a FOIA request to the <a href=\"https://www.foia.gov/?id=076ddcc4-113d-47bf-b00f-0962512a7770&amp;type=component \">Social Security Administration</a></p>\r\n"
```

Comment:
If the summary content has a paragraph over a blue background, this is meant to be styled as a card, and should be created by making the whole paragraph the link text like this:

```
<p><a href="https://secure.ssa.gov/ICON/main.jsp">To find your own 
social security records, contact your local field office. Please
use the field office locator that services your zip code.</a></p>
```

In the app, this markup gets rewritten to the more complex card markup. I haven't looked at the content though, so if the slides have inline links, that's fine; those won't be transformed into cards. See the IRS journey on Dev to see the cards in the summary pages.

But I suspect many of these inline links need to be altered like above.